pipeline/docker/__init__.py

```diff
@@ -0,0 +1,81 @@
+from ntpath import join
+import os
+import yaml
+
+from typing import List
+
+from pipeline.objects import Graph
+
+
+def create_pipeline_api(
```

Comment:
perhaps renaming this function? `dockerize_pipeline`?

also this could be moved directly to a `pipeline/docker.py`  

or this file and now that I think of it, the `decorators.py` could both be moved to `utils`

Reply:
Naming is just a preference here, but happy to move into `docker.py`.

Reply:
Is it an api? I see we create a container, but are we are adding endpoints to send inputs and such? I ask because I see we pull an image for `pipeline-docker` which I don't know

Reply:
Yes it is an API, link [here to the source](https://github.com/mystic-ai/pipeline-docker/blob/main/app/routes/run.py)

Reply:
alright, now i understand, thanks!